addons/payment/models/payment_acquirer.py

```diff
@@ -68,14 +71,21 @@ class PaymentAcquirer(models.Model):
         help="Template for method registration")
     environment = fields.Selection([
         ('test', 'Test'),
-        ('prod', 'Production')], string='Environment',
+        ('prod', 'Production')], string='Mode', help='In test mode, a fake payment is processed through a test '
+                                                     'payment interface. This mode is advised when setting up the '
+                                                     'acquirer. Watch out, test and production modes require '
+                                                     'different credentials.',
         default='test', oldname='env', required=True)
     website_published = fields.Boolean(
-        'Visible in Portal / Website', copy=False,
+        'Activated', copy=False,
         help="Make this payment acquirer available (Customer invoices, etc.)")
+    website_publication_state = fields.Selection(
```

Comment:
This field is useless as the status bar is not clickable to Publish/Unpublish the payment acquirer. Use the website_published field instead.

You can define a new button (like the test/prod) one like this:
``` xml
                    <button class="oe_stat_button" name="website_publish_button"
                        type="object" icon="fa-globe">
                        <field name="website_published" widget="website_button"/>
                    </button>
```
You can also customize the displayed label with the ``options`` attribute.

Reply:
@tivisse 
I tried the folowing but it does not seam to work with the statusbar widget.
<field name="website_published" widget="statusbar"
    options="{'terminology':{'string_true': 'Activated', 'string_false': 'Disabled'}}"/>

Reply:
This should be a stat button instead (class ``oe_stat_button``) on the form view, not a statusbar.

Reply:
originally it was a stat button, but Frédéric asked to change it to a statusbar with a button at the left of the screen. 
https://drive.google.com/file/d/10oXt15xn9yBRvYw8cECt5iHdjEzsdt3Q/view